flask_securest/authorization_providers/role_based_authorization_provider.py

```diff
@@ -0,0 +1,92 @@
+#########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+import os
+import re
+import yaml
+
+from flask import current_app
+
+from flask_securest import rest_security
+from flask_securest.authorization_providers.abstract_authorization_provider\
+    import AbstractAuthorizationProvider
+
+
+REST_SERVICE_HOME_ENV_VAR = 'REST_SERVICE_HOME'
+ALL_METHODS = '*'
+
+
+class RoleBasedAuthorizationProvider(AbstractAuthorizationProvider):
+
+    def __init__(self, roles_config_file_name):
```

Comment:
maybe accept role_loader configuration here and don't make it an abstract entity of flask securest